src/control/system/raft/raft.go

```diff
@@ -437,8 +456,33 @@ func (d *dbData) applyPoolUpdate(op raftOp, data []byte, panicFn func(error)) {
 		panicFn(errors.Errorf("unhandled Pool Service Apply operation: %d", op))
 		return
 	}
+}
 
-	d.MapVersion++
+// applySystemUpdate is responsible for applying the system properties update
```

Comment:
Nit - instead of calling out system properties/attrs in the comment, maybe just call it a system data update.